templates/activities/post_delete.html

```diff
@@ -7,7 +7,7 @@ <h1>Delete this post?</h1>
     {% include "activities/_mini_post.html" %}
     <form action="." method="POST">
         {% csrf_token %}
-        <a class="button" onclick="history.back()">Cancel</a>
+        <button onclick="history.back()" autofocus>Cancel</button>
```

Comment:
This `autofocus` is just for usability improvement similar to common dialog.

<img width="504" alt="Screenshot 2023-01-09 at 2 06 26" src="https://user-images.githubusercontent.com/1425259/211209440-9778a108-3007-4474-96e8-7feac27e2f2a.png">